src/Orchard.Web/Modules/Orchard.Azure.MediaServices/Orchard.Azure.MediaServices.csproj

```diff
@@ -147,6 +147,8 @@
     <Reference Include="System.Xml.Linq" />
   </ItemGroup>
   <ItemGroup>
+    <Content Include="Assets\Css\azuremediaplayer.min.css" />
+    <Content Include="Assets\JavaScript\Lib\azuremediaplayer.min.js" />
```

Comment:
We shouldn't commit minified resources in the repo. Instead commit dev version and let the gulp pipeline handle the minification, as previously agreed.